content/chickpea-salad-sandwich.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Chickpea Salad Sandwiches"
+date: 2021-09-09T10:50:46-07:00
+draft: false
+serves: 4
+total_time_to_make: 15
+time_to_prepare: 15
+time_to_cook: 0
```

Comment:
```suggestion
time_to_cook: 30
```

Reply:
Technically there is no cook time here unless `time_to_cook` is something else?

Reply:
You're right... I saw 2 * 15 which meant 30. The order here is what fooled me. This is correct as is.